test/fixtures/overrides/max-len.fail.ts

```diff
@@ -0,0 +1,13 @@
+/* eslint-disable @typescript-eslint/no-unused-vars */
+/* eslint-disable unicorn/prevent-abbreviations */
+/**
+ * This is a comment that violates the maximum line length we have specified for comments.
```

Comment:
Who is we?

Reply:
fixed with: https://github.com/ridedott/eslint-config/pull/319/commits/267a96360f447175bf63c69e8f81d71b5db5c652